charts/connect/values.yaml

```diff
@@ -2,34 +2,54 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# Configuration settings for the Redpanda Connect Deployment.
 deployment:
+
+  # -- Number of Replicas for the Redpanda Connect Deployment.
   replicaCount: 1
+  # -- Annotations to add to Redpanda Connect Pods.
   podAnnotations: {}
+  # -- Labels to add to Redpanda Connect Pods.
   podLabels: {}
-  # annotations -- Set annotations on the Redpanda Connect Deployment
+  # -- Set Annotations on the Redpanda Connect Deployment.
   annotations: {}
-  # Default 60. ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/
+  # -- Time in seconds for Pod termination grace period.
   terminationGracePeriodSeconds: 60
-  # Rollout the deployment on ConfigMap changes
+  # -- Rollout the deployment on ConfigMap changes.
```

Comment:
I tried testing this but not sure how it works. Could someone confirm what it's supposed to do?

Reply:
If user would update (`helm upgrade`) connect configuration which would only change ConfigMap the Deployment and along with it Pods would not picked up (missing iNotify events probably in Redpanda Connect). That's why config map sha256 sum annotation set in Deployment would help automate config change rollout.
```
  template:
    metadata:
      annotations:
        {{- if .Values.deployment.rolloutConfigMap }}
        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
        {{- end}}
```
https://github.com/redpanda-data/helm-charts/blob/0eab2ffff805416423b7ebec93cb2788fbbeecb5/charts/connect/templates/deployment.yaml#L20-L25

Reply:
But as I see https://github.com/redpanda-data/helm-charts/pull/1594/files#diff-2d6335db92d8ff0df28b4fea97de9c234115fdca59c11fb7a09f7c86275d0539R320 watch option should be mentioned here :) just to cross reference such 'watch' capability.